<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>websocket客户端</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>消息列表</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="url_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>客户端名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="connect_btn">
      <property name="text">
       <string>连接服务器</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>私聊客户端名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dst_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="send_btn">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>url_edit</zorder>
  <zorder>label_2</zorder>
  <zorder>name_edit</zorder>
  <zorder>connect_btn</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>dst_edit</zorder>
  <zorder>send_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
